src/plugins/intel_gpu/tests/test_cases/permute_gpu_test.cpp

```diff
@@ -1902,3 +1908,196 @@ TEST(permute_gpu_f32_dynamic, bfyx_0_2_3_1) {
         EXPECT_FLOAT_EQ(answers[i], output_ptr[i]);
     }
 }
+
+class permute_bfzyx_to_bfyxz: public TiledPermuteTest {};
+
+INSTANTIATE_TEST_SUITE_P(, permute_bfzyx_to_bfyxz,
+    ::testing::ValuesIn(std::vector<TiledPermuteParam> {
+            {{1, 3, 85, 20, 20}, format::bfzyx},
+            {{1, 3, 85, 40, 40}, format::bfzyx},
+            {{1, 3, 85, 80, 80}, format::bfzyx}
+        }));
+
+TEST_P(permute_bfzyx_to_bfyxz, combined) {
+    auto p = GetParam();
+    run_test<cldnn::data_types::f32>(p.sizes, p.format_fsv, "permute_bfzyx_to_bfyxz", {0, 1, 3, 4, 2});
```

Comment:
maybe it worth to add another format types if they are supported by this new kernel